gahtc/gahtc/urls.py

```diff
@@ -28,6 +29,7 @@
 
 urlpatterns = [
 	url(r'^', include(website_urls)),
+    url(r'^mapbuilder/',  include(mapbuilder_urls)),
```

Comment:
We are missing the menu link pointing to this url in the main application.
![screen shot 2018-02-20 at 15 14 50](https://user-images.githubusercontent.com/1008418/36423388-def81982-1650-11e8-98fa-9cf7610460c8.png)